Add perm link (make title) to sample entries in README file (#9)
microsoft/OpenXR-Unity-MixedReality-Samples: README.md

# OpenXR + Unity + Mixed Reality Samples

## Welcome!

![OpenXR-Unity-MixedReality-Samples-MainMenu](Readme/OpenXR-Unity-MixedReality-Samples-MainMenu.jpg)

These sample projects showcase how to build Unity applications 
for HoloLens 2 or Mixed Reality headsets using the Mixed Reality OpenXR plugin.
For more details on installing related tools and setting up a Unity project,
please reference [the plugin documentation on https://docs.microsoft.com/](https://aka.ms/openxr-unity).

> NOTE: This repository uses [Git Large File Storage](https://git-lfs.github.com/) to store large files,
such as Unity packages and images. Please install the latest git-lfs before cloning this repo.

> NOTE: Features using AR Raycasts and AR Planes require the latest preview OpenXR runtime,
which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions). 
To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.

> ### Sample for Anchors and anchor persistence

Reviewer: nit: Sorry, I think I forgot to post this comment on the last review.
These first four "Sample for ..." headers have some inconsistent capitalization. I think sentence casing would match the rest of our documentation.
```suggestion
### Sample for anchors and anchor persistence
```

Welcome!

> ![OpenXR-Unity-MixedReality-Samples-MainMenu](Readme/OpenXR-Unity-MixedReality-Samples-MainMenu.jpg)

Reviewer: It looks like this new image is a lower resolution - do you have a higher-resolution image available? The text in the buttons is difficult to read now, and I think it used to function as a nice visual overview of what scenes the BasicSample provides.
https://github.com/microsoft/OpenXR-Unity-MixedReality-Samples/blob/yl/update_readme/Readme/OpenXR-Unity-MixedReality-Samples-MainMenu.jpg

If you want, I could also re-take this photo in the same resolution the image on main is.

Author: fixed.

> These sample projects showcase how to build Unity applications for HoloLens 2 or Mixed Reality headsets using OpenXR plugin. For more details on installing related tools and setup Unity project, please reference [https://docs.microsoft.com/](https://aka.ms/openxr-unity).

Reviewer: nit: I'd previously used the full name "Mixed Reality OpenXR Plugin", since searching for "OpenXR plugin" gives Unity's plugin as the first result and this more specific plugin as the second one. If you think this is fine, then it's probably not a big issue.
```suggestion
These sample projects showcase how to build Unity applications for HoloLens 2 or Mixed Reality headsets using the Mixed Reality OpenXR plugin. For more details on installing related tools and setting up a Unity project, please reference [the plugin documentation on https://docs.microsoft.com/](https://aka.ms/openxr-unity).
```

Author: fixed

> > NOTE: This repository uses [Git Large File Storage](https://git-lfs.github.com/) to store large files, such as unity package and images. Please install the latest git-lfs before cloning this repo.

Reviewer: super nit:
```suggestion
> NOTE: This repository uses [Git Large File Storage](https://git-lfs.github.com/) to store large files, such as Unity packages and images. Please install the latest git-lfs before cloning this repo.
```

Author: fixed
